packages/mobile/src/components/remix-carousel/RemixCarousel.tsx

```diff
@@ -9,6 +9,7 @@ import { RemixContestCard } from './RemixContestCard'
 
 export type RemixContestCardProps = {
   trackIds: ID[] | undefined
+  inView?: boolean
```

Comment:
unused?

Reply:
good catch, im going to drop this component and put everything in remix-section